engine/src/juliabox/db/api_spec.py

```diff
@@ -16,7 +16,7 @@ class JBoxAPISpec(JBoxDB):
     ]
 
     INDEXES = [
-        GlobalKeysOnlyIndex('publisher-api_name-index', parts=[
+        GlobalKeysOnlyIndex('publisher_api_name_index', parts=[
```

Comment:
The name is actually immaterial, but since column names could contain `_`, and index names were made of column names, `-` was used as a separator.
Any restrictions in having `-` in the index name?